sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c

```diff
@@ -286,7 +287,7 @@ void ms45xx_i2c_event(void)
       float temp = ms45xx.temperature / 10.0f;
       AbiSendMsgTEMPERATURE(MS45XX_SENDER_ID, temp);
       // Compute airspeed
-      ms45xx.airspeed = sqrtf(Max(ms45xx.pressure * ms45xx.airspeed_scale, 0));
+      ms45xx.airspeed = sqrtf(Max(ms45xx.pressure, 0)) * MS45XX_AIRSPEED_SCALE;
 
 #if USE_AIRSPEED_MS45XX
       stateSetAirspeed_f(ms45xx.airspeed);
```

Comment:
Is it a good thing that drivers push to the state directly or should we enforce pushing through air_data?

Reply:
I would be in favor of an ABI message to go through air_data module.

Reply:
removed the ```stateSetAirspeed```, checked all airframes using this sensor that it has the ```<air_data>``` module. Removed all ```USE_MS45XX`` and also removed the ```MS45XX_SYNC_SEND``` since it now is sent via ```telemetry.xml``` and fixed some docs.